SimG4Core/PhysicsLists/python/physicsQGSP_BERT_G4104_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from SimG4Core.Configuration.SimG4Core_cff import *
+
+g4SimHits.Physics.EmaxBERT   = cms.double(8.)  # in GeV
```

Comment:
@bsunanda , this 6-8 GeV transition is not justified by our previous testing. Do we really need playing energy transition again? But that we have to go via many combinations, I do not see how we make this choice.

Reply:
This was the transition region in Geant4.10.4.p03 - I just want the HGCal TB team to see how bad/good it is